src/trackers/helpers/getOwner.js

```diff
@@ -0,0 +1,17 @@
+const sharedData = require('./sharedData.js')
+
+function getOwner (domain) {
+    if (!domain) {
+        return
+    }
+    
+    if (sharedData.entityMap.get(domain)) {
```

Comment:
Getting it twice is a bit weird. Maybe let's use `has`?

```suggestion
    if (sharedData.entityMap.has(domain)) {
```

Or maybe

```suggestion
   const owner = sharedData.entityMap.get(domain);
   
   if (owner) {
      return owner;
   }
```